app/src/main/res/xml-v25/shortcuts.xml

```diff
@@ -7,7 +7,7 @@
         android:shortcutLongLabel="@string/favorite_title">
         <intent
             android:action="android.intent.action.VIEW"
-            android:targetPackage="io.github.droidkaigi.confsched2018.debug"
+            android:targetPackage="io.github.droidkaigi.confsched2018"
```

Comment:
it looks like it should be corrected in the `release` flavour instead maybe 🤔  

https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/release/res/xml-v25/shortcuts.xml#L10

Reply:
🤔

Reply:
Thanks we should change that 👍